src/fitnesseMain/badCodeToTestCheckstyle.java

```diff
@@ -0,0 +1,63 @@
+package fitnesseMain;
+
+import java.util.*; //BAD: star imports
+
+class  Hello
+{
+  public final static String hello = "helloWorld!"; //BAD: modifier order
+
+  public String badCommaWhiteSpaceBeforeAndAfter(String input ,String anotherInput)
+    {
+        return "badCommaWhiteSpaceBeforeAndAfter";
+    }
+
+    public void badGenericsWhiteSpace()
+    {
+      List< String> strings = Arrays.asList("foo", "bar");
+      System.out.print(strings.get(0));
+    }
+
+    public boolean badWhiteSpaces()
+    {
+        int sum = 2+2;
+        int diff = 2-1;
+        System.out. println("hello");
+        return sum+diff==5;
+    }
+
+    @SafeVarargs public final void badAnnotations(String... badVarargsAnnotation)
+    {
+        @SuppressWarnings("default") Hello badSuppressWarningsAnnotation;
+    }
+    
+    public void badArrayTypeStyle()
+    {
+      String args[] = {"hello", "world"};
+    }
+    
+    public void badBraces(){
+      System.out.println("bad");
+      System.out.println("braces!");}
+      
+    public void badMultipleStringLiterals()
+    {
+      //TODO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Comment matches to-do format '.*'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3ACheck_Any_Comments)